adapters/gamma/gammatest/supplemental/bad-request.json

```diff
@@ -0,0 +1,51 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "app":{
+    	"id":"test-app-id",	
+    	"name":"test-app-name",
+    	"bundle":"test-app-bundle"
+    },
+
+    "device":{
+    	"ua":"test-device-ua",
+    	"ip":"test-device-ip",
+      "ifa":"test-device-ifa",
```

Comment:
Just a nit pick but since there are other changes, can you please fix the formatting in this file? The spaces seem to be off at a couple places